### ISSUE TITLE
Error: length=2; index=10

### ISSUE THREAD

**Issue by KatarinaSimnova** [corymsmith/react-native-fabric#108]
After running `Crashlytics.crash();` using my android device I get this error:
![16930757_1567561266590427_798522080_o](https://cloud.githubusercontent.com/assets/11189150/23217061/506be5c8-f918-11e6-9246-3b98c3dafc01.png)

**Comment by corymsmith** [comment]
Yes, this is how it crashes so that you can verify that crashes are being sent. This is expected behavior, what are you trying to do?

**Comment by KatarinaSimnova** [comment]
The problem is that crash message wasn't sent. 
Ok, now I know that something else caused it. 
Probably bad configuration. Thank you for your quick response.

**Comment by KatarinaSimnova** [comment]
Are you sure your Manual installation guide in README is up to date?

**Comment by corymsmith** [comment]
What issue are you having?

**Comment by SamaneM** [comment]
i'm facing this problem(crash message doesn't send) , may you tell me what was the solution ?  @KatarinaSimnova
